docs/pages/kubernetes-access/helm/guides/gcp.mdx

```diff
@@ -158,7 +157,6 @@ Go back to the "Service Accounts" view in Google Cloud IAM & Admin.
   ![Private key saved](../../../../img/helm/gcp/12-privatekey@1.5x.png)
 </Figure>
 
-
```

Comment:
This step below:

```
helm install cert-manager jetstack/cert-manager \
   --create-namespace \
   --namespace cert-manager \
   --set installCRDs=true
```

1. `error: unable to recognize "gcp-issuer.yaml": no matches for kind "Issuer" in version "cert-manager.io/v1"`
2. Also:
```
Error: rendered manifests contain a resource that already exists. 

Unable to continue with install: could not get information about the resource: mutatingwebhookconfigurations.admissionregistration.k8s.io "cert-manager-webhook" is forbidden: 
User "x@example.com" cannot get resource "mutatingwebhookconfigurations" in API group "admissionregistration.k8s.io" at the cluster scope: GKEAutopilot authz: cluster scoped resource "mutatingwebhookconfigurations/" is managed and access is denied
```

Reply:
Everything up to this point works smoothly. I think this is Helm-specific. The failure to find the `Issuer` means that the jetstack/cert-manager wasn't installed.